libs/designsystem/src/lib/components/radio/radio-group/radio-group.component.ts

```diff
@@ -20,9 +23,16 @@ import { RadioComponent } from '../radio.component';
   selector: 'kirby-radio-group',
   templateUrl: './radio-group.component.html',
   styles: ['ion-radio-group { display: inherit; flex-wrap: inherit}'],
+  providers: [
+    {
+      provide: NG_VALUE_ACCESSOR,
+      useExisting: RadioGroupComponent,
+      multi: true,
```

Comment:
I'm missing set error logic. I havent figured out yet how to identify error state.

Reply:
@jakobe and I resolved this yesterday. Essentially, angular sets a .error class on the form control element, which we can then use to set the color of all radios in the radio group.